### DISCUSSION TITLE
Filament runout sensor

### DISCUSSION THREAD

**Discussion by gerthalberg** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1480] [Q&A]
It seems like when you run the tft in Marlin mode there will be a reaction when the filament runout switch is triggered. This is not the case when running in BTT/touch mode - unless the runout switch is connected to the tft. The question is would it be possible to have the same switch connected to both the board and the TFT at the same time? probably with a y-splitter. My board is SKR 1.4 Turbo and TFT is TFT35 E3 V3.

**Comment by oldman4U** [comment]
According to kisslorand, you have to add a "M75" to your start gcode of the slicer or the TFT to make this work. But this is less a discussion than an Issue and you should check existing open and closed issues first.

Hope this helps

**Reply by gerthalberg** [reply]
First of all thank's for the reaction.
I have already activated the #define PRINTJOB_TIMER_AUTOSTART to get Marlin to even react to the filament runout switch and it works in Marlin mode - So am I correct in understanding that what you're saying is if Print job timer is running the TFT in touch mode will react to a filament runout event even if the switch is not connected to the TFT runout plug? Will now of-course check per your suggestion.

**Reply by gerthalberg** [reply]
A little funny putting in an answer to myself - but here it comes. Did read through the issue list as suggested - there is no solution though. There is mention from kisslorand about the M75 command but as written earlier this does not apply since it merely has to be there to get Marlin to react correctly on flamet runout. I will try to run an extra provisional wire from the signal pin to the tft - hope I don't let the smoke out of my TFT.

**Comment by kisslorand** [comment]
Don't do that!!!
It's a long story why not to but I had to warn you ASAP!

**Reply by gerthalberg** [reply]
Once again - thank's for the answer and probably saving me the grief. I will NOT run a provisional wire I will instead test tomorrow and verify that the TFT indeed reacts to the Marlin event. I will read up to see, if the host service/thing needs to be activated in Marlin.:-)

**Reply by 3dfixer** [reply]
did you get this solved?
was thinking maybe got to put twin runout sensors in  :-)

**Comment by 3dfixer** [comment]
what board have you got?

**Reply by gerthalberg** [reply]
Haven't done the firmware upgrade yet. The board is an SKR 1.4 Turbo.

**Reply by oldman4U** [reply]
Which firmware update. You should add a M75 to the start gCode and nothing else.

**Reply by 3dfixer** [reply]
i've spent days and days(i'm currently furloughed so go plenty of time) messing about with this.
you have to go chasing different bits of info everywhere
i've got skr 1.4   tmc2209 drivers bltouch probe tft35v3.0 3 pin runout sensor on e0det
just got about everything working, if you want my configs let me know, should just be a 
case of altering board name
